source/PlayniteSDK/Models/Game.cs

```diff
@@ -2028,6 +2073,11 @@ public List<GameField> GetDifferences(Game otherGame)
                 changes.Add(GameField.PlayCount);
             }
 
+            if (InstallSize != otherGame.InstallSize)

```

Comment:
Missing for `LastSizeScanDate`.

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3017/commits/220778689267ab2f0d2aa9adc43ab38030dcf731